react/components/atoms/container-swipeable/container-swipeable.js

```diff
@@ -60,27 +60,32 @@ export class ContainerSwipeable extends PureComponent {
         return this.state.containerHeightLoaded && this.state.headerHeightLoaded;
     };
 
+    overlayVisible = () => {
+        return this.isLoaded() && this.state.visible;
+    };
+
     open() {
         if (this.animating) return;
 
         this.animating = true;
 
-        Animated.parallel([
-            Animated.timing(this.state.contentHeight, {
-                toValue: 1,
-                duration: this.props.animationsDuration,
-                easing: Easing.inOut(Easing.ease)
-            }),
-            Animated.timing(this.state.overlayOpacity, {
-                toValue: 0.5,
-                duration: this.props.animationsDuration,
-                useNativeDriver: true,
-                easing: Easing.inOut(Easing.ease)
-            })
-        ]).start(() => {
-            this.animating = false;
-            this.setState({ visible: true }, () => {
-                this.props.onVisible(true);
+        this.setState({ visible: true }, () => {

```

Comment:
This makes sense, it was a mistake from me to have written it the other way around. The logic is: if closing goes animate -> setState, then opening should be the other way around (setState -> animate). It's a bit like opening/closing resources, where you always do stuff in the inverse order.

Reply:
In any case, what problem was this causing? I'm curious.

Reply:
The animation wouldn't work